raiden/network/transport/matrix/client.py

```diff
@@ -253,8 +251,15 @@ def __init__(
         self.sync_iteration = 0
 
         self._sync_filter_id: Optional[int] = None
+        self._sync_filter_id_to_filter: Dict[int, Dict[str, Any]] = {}
+
+    @property
+    def _sync_filter(self) -> Optional[Dict[str, Any]]:
+        if self._sync_filter_id and self._sync_filter_id in self._sync_filter_id_to_filter:
+            return self._sync_filter_id_to_filter[self._sync_filter_id]
+        return None
```

Comment:
Unless I'm missing something it seems the only reason to have the dict and that method is so we can log the content of the filter during sync.

I'd say it would make more sense to log the filter content and assigned id at the time they are created and then simply log the filter id during sync.